pub/css/main.css

```diff
@@ -96,21 +96,109 @@ textarea {
 /* ==========================================================================
    Author's custom styles
    ========================================================================== */
+table {
+    width: 90%;
+}
 
+    thead th {
+        border-bottom: 1px solid black;
+        text-align: center;
+    }
+    tbody th {
+        border-right: 1px solid black;
+        text-align: right;
+    }
+    tbody td {
+        width: 10%;
+        padding-bottom: 10%;
+        border: 1px solid black;
+    }
 
+    tbody :nth-child(even) td:nth-child(even),
+    tbody :nth-child(odd) td:nth-child(odd){
+        background-color: darkgreen;
+    }
 
+:before {
+    font-size: 2em;
+}
 
+/*Black Pieces*/
 
+/*rooks*/
+#A8:before, #H8:before {
+    position: absolute;
+    content: "\265C";
+}
 
+/*knights*/
+#B8:before, #G8:before {
+    position: absolute;
+    content: "\265E"
+}
 
+/*bishops*/
+#C8:before, #F8:before {
+    position: absolute;
+    content: "\265D";
+}
 
+/*queen*/
+#D8:before {
+    position: absolute;
+    content: "\265B";
+}
 
+/*king*/
+#E8:before {
+    position: absolute;
+    content: "\265A";
+}
 
+/*pawns*/
+#A7:before, #B7:before, #C7:before, #D7:before, #E7:before, #F7:before, #G7:before, #H7:before {
+    position: absolute;
+    content: "\265F";
+}
+
+
+
+/*white pieces*/
+/*rooks*/
+#A1:before, #H1:before {
+    position: absolute;
+    content: "\2656";
+}
 
+/*knights*/
+#B1:before, #G1:before {
+    position: absolute;
+    content: "\2658";
+}
 
+/*bishops*/
+#C1:before, #F1:before {
+    position: absolute;
+    content: "\2657";
+}
 
+/*queen*/
+#D1:before {
+    position: absolute;
+    content: "\2655";
+}
 
+/*king*/
+#E1:before {
+    position: absolute;
+    content: "\2654";
+}
 
+/*pawns*/
+#A2:before, #B2:before, #C2:before, #D2:before, #E2:before, #F2:before, #G2:before, #H2:before {
+    position: absolute;
+    content: "\2659";
+}
```

Comment:
Haven't gotten to your later submissions yet, but I hope you've come to a better solution now... adding pawns via `#id` references is the most tedious, for sure. Using a `.pawn` class would be more efficient.